### ISSUE TITLE
Wrong link in README.md

### ISSUE THREAD

**Issue by kxkv** [src-r-r/ink-in-time#5]
To book an appointment, the following link is given:
https://damngood.tech/schedule/60min/

However, that link is not working.

**Comment by src-r-r** [comment]
Yes, thank you. I was aware this was the case. I've had to make a few changes to my websites while also reaching out find new clients so I've been pretty busy lately. I'll hopefully have a working site up soon.

I found recent improvements I was making to the site weren't working so I'm redesigning it to include a postgres database.

I do thank you for opening an issue. Sometimes I wonder if anyone is interested in my projects. 😆 I'll hopefully have this one closed soon.

**Comment by pepper3k** [comment]
> I do thank you for opening an issue. Sometimes I wonder if anyone is interested in my projects. 😆 I'll hopefully have this one closed soon.

I'm interested. Very much. If you need any help you can count on me. I'm officially volunteering myself.

**Comment by sysfu** [comment]
Is there an active booking link where users can demo the software?

**Comment by pepper3k** [comment]
> Is there an active booking link where users can demo the software?

I wouldn't mind setting it up, if anyone has some shared hosting they can spare

**Comment by src-r-r** [comment]
Thanks for all the input folks! This is great to hear. I feel very encouraged!

Sorry @sysfu , currently there's no demo. I did have one up on my website, but because of my failure to find enough substantial work, I haven't been able to afford to host it myself.

If anyone is wanting to host it for themselves for their own site, though, I'd love to hop on a call to help. It's been a while since I touched this project, but I'd love to get it revitalized.

I'd love any feedback from folks.

Love the support everyone!

**Comment by sysfu** [comment]
> Thanks for all the input folks! This is great to hear. I feel very encouraged!
> 
> Sorry @sysfu , currently there's no demo. I did have one up on my website, but because of my failure to find enough substantial work, I haven't been able to afford to host it myself.
> 
> If anyone is wanting to host it for themselves for their own site, though, I'd love to hop on a call to help. It's been a while since I touched this project, but I'd love to get it revitalized.
> 
> I'd love any feedback from folks.
> 
> Love the support everyone!

Sure, I have some lightly used VPS that I could host it on.  Please send me the hosting requirements and let's get a working demo up for people to try.

**Comment by pepper3k** [comment]
> > Thanks for all the input folks! This is great to hear. I feel very encouraged!
> > Sorry @sysfu , currently there's no demo. I did have one up on my website, but because of my failure to find enough substantial work, I haven't been able to afford to host it myself.
> > If anyone is wanting to host it for themselves for their own site, though, I'd love to hop on a call to help. It's been a while since I touched this project, but I'd love to get it revitalized.
> > I'd love any feedback from folks.
> > Love the support everyone!
> 
> Sure, I have some lightly used VPS that I could host it on. Please send me the hosting requirements and let's get a working demo up for people to try.

@sysfu if you need any help setting it up, count me in.
